openpype/hosts/nuke/api/plugin.py

```diff
@@ -820,12 +820,12 @@ def __init__(self,
         if ".{}".format(self.ext) not in VIDEO_EXTENSIONS:
             # filename would be with frame hashes if
             # the file extension is not in video format
-            filename = get_head_filename_without_hashes(
+            filename = prepend_name_before_hashed_frame(
```

Comment:
I still think the code is weird.

The class inherits from [`ExporterReview`](https://github.com/ynput/OpenPype/blob/4a2417d2ca741c6b5a8db81b042e0bfff7a4d12a/openpype/hosts/nuke/api/plugin.py#L559) which already computes `self.fhead` like is done outside of this logic to define `self.fhead + self.name + ...` etc.

Doesn't that just mean we should somehow be fixing the logic [here](https://github.com/ynput/OpenPype/blob/4a2417d2ca741c6b5a8db81b042e0bfff7a4d12a/openpype/hosts/nuke/api/plugin.py#L585-L609)?

What's the reason we can't rely on that `self.fhead`? Doesn't this just mean that the design of `ExporterReview` class is buggy to begin with and doesn't support sequences (or at least not paths with hashes - and if it's just about the hashes, _why_ are we passing the filepath with hashes to begin with).